docs/conf.py

```diff
@@ -21,6 +21,7 @@
     "sphinx.ext.intersphinx",
     "sphinx.ext.autosectionlabel",
     "sphinx.ext.napoleon",
+    "sphinx_autodoc_typehints",
```

Comment:
When adding a new Sphinx extension, it should be documented where appropriate. In this case, update https://icalendar.readthedocs.io/en/latest/contribute/development.html#apidoc with a new typehint docstring example.

Reply:
Oh. That is just there so that the type aliases get resolved.
The docstring there is typed already. What would you like to see? At the moment, I do not know how to follow your request.

Reply:
No, it is duplicated code to type the arguments inside the string. This is the solution to resolving TypeAlias variables to their actual value:

https://icalendar--941.org.readthedocs.build/en/941/reference/api/icalendar.html#icalendar.Calendar.new
<img width="600" height="53" alt="grafik" src="https://github.com/user-attachments/assets/790c1f15-a1a6-4cea-89bf-97fd8f85b6c6" />

It seems they are automatically embedded and resolved.